openvdb_points/tools/AttributeSet.cc

```diff
@@ -442,7 +442,7 @@ AttributeSet::readAttributes(std::istream& is)
     AttrArrayVec(mDescr->size()).swap(mAttrs); // allocate vector
 
     for (size_t n = 0, N = mAttrs.size(); n < N; ++n) {
-        mAttrs[n] = AttributeArray::create(mDescr->type(n), 1);
+        mAttrs[n] = AttributeArray::create(mDescr->type(n), 1, 1);
```

Comment:
A comment to say that the correct stride is set in the attribute array deserialization would be handy